stable/instana-agent/templates/agentsecret.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "instana-agent.baseName" . }}-agent-secret
+  name: {{ template "instana-agent.name" . }}-agent-secret
```

Comment:
Resource names should be based on the fullname template. In general, I'd suggest not to add suffixes unless you have multiple resources of the same kind.

Reply:
In this ^ case, we have 2 `Secret` resources, one agentsecret another downloadsecret.

But for the others, I can remove the suffix.